### PR TITLE
closes issue #52

### PR DESCRIPTION
**Pull request by smurfik**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -58,18 +60,22 @@ def name
...
+  scope :search_by_distance, (lambda { |user, distance|
       if user.present? && distance.present?
         self.near([user.latitude, user.longitude], distance)
             .where.not(id: user.id)
       end
   })
 
   def full_street_address
-    return "#{street} #{city}, #{state} #{zipcode}"
+    return "#{full_street} #{city}, #{state} #{zipcode}"
```
Comment (seanmarcia) [inline app/models/user.rb]: why the explicit return? (I realize this was already in before)

File: app/models/user.rb [hunk 2]
```diff
@@ -58,18 +60,22 @@ def name
...
+  end
+
+  def full_street
+    [street1, street2, street3].join(" ").strip
   end
 
   def full_street_address_changed?
-    return street_changed? || city_changed? || state_changed? || zipcode_changed?
+    return street1_changed? || street2_changed? || street3_changed? || city_changed? || state_changed? || zipcode_changed?
```
Comment (seanmarcia) [inline app/models/user.rb]: Another explicit return :)

**Comment by smurfik** [comment]
oops :) fixed it

**Comment by smurfik** [comment]
merged master in -- google api failures

**Pull request merged**